tests/common/osutil/test_default.py

```diff
@@ -643,327 +639,6 @@ def test_conf_sudoer(self, mock_dir):
         print("WRITING TO {0}".format(waagent_sudoers))
         self.assertEqual(1, count)
 
-    @staticmethod
```

Comment:
moved to test_firewall_manager, or deleted, since some of those test don't make sense for the new code